vidarr-core/src/main/java/ca/on/oicr/gsi/vidarr/core/LocalOutputProvisioner.java

```diff
@@ -0,0 +1,100 @@
+package ca.on.oicr.gsi.vidarr.core;
+
+import static ca.on.oicr.gsi.vidarr.OperationAction.load;
+import ca.on.oicr.gsi.Pair;
+import ca.on.oicr.gsi.status.SectionRenderer;
+import ca.on.oicr.gsi.vidarr.BasicType;
+import ca.on.oicr.gsi.vidarr.OperationAction;
+import ca.on.oicr.gsi.vidarr.OutputProvisionFormat;
+import ca.on.oicr.gsi.vidarr.OutputProvisioner;
+import ca.on.oicr.gsi.vidarr.OutputProvisionerProvider;
+import ca.on.oicr.gsi.vidarr.core.LocalOutputProvisioner.ProvisionState;
+import ca.on.oicr.gsi.vidarr.core.LocalOutputProvisioner.PreflightState;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.stream.Stream;
+import java.util.zip.CRC32;
+import javax.xml.stream.XMLStreamException;
+
+public class LocalOutputProvisioner implements OutputProvisioner<PreflightState, ProvisionState> {
+
+  public record PreflightState(){}
+  public record ProvisionState (String inputFile,
+                               JsonNode metadata,
+                               String vidarrId){
+    public OutputProvisioner.Result move() throws IOException {
+      final Path inputPath = Paths.get(inputFile),
+          outputPath = Paths.get(metadata.get("outputDirectory").asText()),
+          newPath = Files.move(inputPath, outputPath.resolve(inputPath.getFileName()), StandardCopyOption.REPLACE_EXISTING);
+      // Calculate CRC32
+      byte[] data = Files.readAllBytes(newPath);
```

Comment:
Could you clarify in the markdown file that this output provisioner should only be used for small text files?

Reply:
Why? It'll move anything

Reply:
The [javadoc for readAllBytes](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllBytes-java.nio.file.Path-) says "Note that this method is intended for simple cases where it is convenient to read all bytes into a byte array. It is not intended for reading in large files." Have you tested this out on a file that's larger than 2GB, or larger than 100+GB? If so, did you run into issues with byte array size limits, or memory issues, or locking up Vidarr on this operation?

Reply:
Nope. I'll update the docs.